tests/by-util/test_hashsum.rs

```diff
@@ -244,7 +244,7 @@ fn test_check_file_not_found_warning() {
         .arg(at.subdir.join("testf.sha1"))
         .fails()
         .stdout_is("sha1sum: testf: No such file or directory\ntestf: FAILED open or read\n")
-        .stderr_is("sha1sum: warning: 1 listed file could not be read\n");
+        .stderr_is("sha1sum: WARNING: 1 listed file could not be read\n");
```

Comment:
The code potentially outputs a plural form. Is that part of our guarantee to the user?
- If yes, then it should be part of the tests.
- If no, then the code can be deleted/simplified.

Reply:
i will do it in a different PR, I have other changes pending